packages/pluggableWidgets/video-player-native/src/__tests__/VideoPlayer.spec.tsx

```diff
@@ -81,6 +86,6 @@ describe("VideoPlayer", () => {
         fireEvent(component.getByTestId("video-player-test"), "error");
 
         expect(component.UNSAFE_getByType(Text).props.style).toEqual({ color: "white" });
-        expect(component.UNSAFE_getByType(Text).props.children).toEqual("The video failed to load :(");
+        expect(component.UNSAFE_getByType(Text).props.children).toEqual("The video failed to load");
     });
 });
```

Comment:
It would be good to add a unit test for the new functionality, namely full screen

Reply:
done